tensorflow_serving/tools/docker/Dockerfile.mkl

```diff
@@ -0,0 +1,98 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG TF_SERVING_VERSION=latest
+ARG TF_SERVING_BUILD_IMAGE=tensorflow/serving:${TF_SERVING_VERSION}-devel-mkl
+
+FROM ${TF_SERVING_BUILD_IMAGE} as build_image
+FROM ubuntu:18.04
+
+ARG TF_SERVING_VERSION_GIT_BRANCH=master
+ARG TF_SERVING_VERSION_GIT_COMMIT=head
+
+LABEL maintainer="Karthik Vadla <karthik.vadla@intel.com>"
+LABEL tensorflow_serving_github_branchtag=${TF_SERVING_VERSION_GIT_BRANCH}
+LABEL tensorflow_serving_github_commit=${TF_SERVING_VERSION_GIT_COMMIT}
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        ca-certificates htop \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install TF Serving pkg
+COPY --from=build_image /usr/local/bin/tensorflow_model_server /usr/bin/tensorflow_model_server
+
+# Install MKL libraries
```

Comment:
This works, but I'm curious if there's a more standard approach to installing mkl libraries (say, via APT?)

Reply:
We don't have `APT` packages to install `mkl`. https://github.com/intel/mkl-dnn#installation
Either should be compiled or should be downloaded from here https://github.com/intel/mkl-dnn/releases

Reply:
@karthikvadla is correct. `MKL` support in Tensorflow is always built from source.

Reply:
Sounds good to me.